src/greatwall/resources/knowledge/fractal.py

```diff
@@ -168,16 +177,24 @@ def mandelbrot_set(
         x = np.linspace(x_min, x_max, width)
         y = np.linspace(y_min, y_max, height)
 
+        self.computed_values = {}
```

Comment:
The same as the above comment.

Reply:
I left the variable initialization in the constructor. Thank you.